src/Containers/CadView.jsx

```diff
@@ -373,7 +378,10 @@ const CadView = () => {
               viewer={viewer}
               element={selectedElement}
               close = {()=>setShowItemPanel(false)}
-              topOffset = {PANEL_TOP}/>
+              topOffset = {PANEL_TOP}
+              placeCutPlane = {()=>placeCutPlane()}
```

Comment:
Not necessary here, but we need to think of a quick way to factor these out of CadView, for improved latency.